### PR TITLE
Move implementation to `src` dir

### PR DESCRIPTION
**Pull request by solid-daniilmarchenko**

### CODE REVIEW

**Comment by solid-daniilmarchenko** [comment]
i've got a bunch of errors while testing the example app. doesn't look like my update's fault, but should be fixed in a separate PR

image

**Comment by solid-daniilmarchenko** [comment]
Fixed in https://github.com/solid-software/languagetool_textfield/pull/78

File: .github/workflows/code_check.yaml [hunk 1]
```diff
@@ -23,24 +23,14 @@ jobs:
...
-          if [ -d example ]; then dirs_to_analyze+=" example"; fi
-          if [ dirs_to_analyze != "" ]
-          then
-            dart run dart_code_metrics:metrics \
-              analyze \
-              $dirs_to_analyze \
-              --fatal-warnings \
-              --fatal-performance \
-              --fatal-style
-          fi
```
Comment (solid-daniilmarchenko) [inline .github/workflows/code_check.yaml:43]: @solid-yuriiprykhodko looks ok?

Comment (yurii-prykhodko-solid) [reply .github/workflows/code_check.yaml:43]: solid_lints no longer uses DCM. 

Add `dart run custom_lint`.

**Review by solid-yuriiprykhodko — APPROVED** [review]
LGTM

**Pull request merged**